libnavigation-core/src/test/java/com/mapbox/navigation/core/fasterroute/FasterRouteDetectorTest.kt

```diff
@@ -37,7 +56,7 @@ class FasterRouteDetectorTest {
     }
 
     @Test
-    fun shouldDefaultToFalseWhenDurationIsNull() {
+    fun shouldDefaultToFalseWhenDurationIsNull() = runBlocking {
```

Comment:
The answer is in, runBlocking is better than runBlockingTest

https://github.com/Kotlin/kotlinx.coroutines/issues/1222#issuecomment-528787927